### ISSUE TITLE
Issue with Simplecov

### ISSUE THREAD

**Issue by RickCSong** [Shopify/bootsnap#35]
Hey!  Wasn't sure if already know, but we're seeing an issue with the iseq cache for Simplecov

```
$ rspec
Enabling Bootsnap
Coverage report generated for RSpec to /Users/rickcsong/Development/myapp/coverage. 0.0 / 0.0 LOC (100.0%) covered.
/Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/compile_cache/iseq.rb:8:in `to_binary': should not compile with coverage (RuntimeError)
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/compile_cache/iseq.rb:8:in `input_to_storage'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/compile_cache/iseq.rb:30:in `fetch'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/compile_cache/iseq.rb:30:in `load_iseq'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/bootsnap-0.2.14/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
  from /Users/rickcsong/Development/myapp/spec/rails_helper.rb:11:in `<top (required)>'
  from /Users/rickcsong/Development/myapp/spec/models/dummy.rb:1:in `require'
  from /Users/rickcsong/Development/myapp/spec/models/dummy.rb:1:in `<top (required)>'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/configuration.rb:1435:in `load'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/configuration.rb:1435:in `block in load_spec_files'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/configuration.rb:1433:in `each'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/configuration.rb:1433:in `load_spec_files'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/runner.rb:100:in `setup'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/runner.rb:86:in `run'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/runner.rb:71:in `run'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/lib/rspec/core/runner.rb:45:in `invoke'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/gems/rspec-core-3.5.4/exe/rspec:4:in `<top (required)>'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/bin/rspec:23:in `load'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/bin/rspec:23:in `<main>'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
  from /Users/rickcsong/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by burke** [comment]
Yeah -- I should probably add this to the readme, but the ruby VM explicitly disallows iseq loading/dumping when coverage is enabled. I looked for a workaround and couldn't find anything.

**Comment by burke** [comment]
https://github.com/ruby/ruby/blob/trunk/compile.c#L8519-L8521 fwiw

**Comment by burke** [comment]
Just released 0.3.2, which, though it still can't optimize this case, doesn't actively blow up.

**Comment by jules2689** [comment]
https://github.com/ruby/ruby/blob/52de8260968cf77dda5980e632c94900f1487428/compile.c#L8545

Is the direct link with commit.
